lib/vimeo_api/vimeo_wrapper.rb

```diff
@@ -0,0 +1,36 @@
+module VimeoWrapper
+
+  VIMEO_MAX_WIDTH = "500px"
+  VIMEO_MAX_HEIGHT = "450px"
+
+  def vimeo_auth
+    "bearer #{vimeo_access_token}"
+  end
+
+  def vim_base_uri
+    "https://api.vimeo.com"
+  end
+
+  def vimeo_access_token
+    ENV["VIMEO_ACCESS_TOKEN"]
+  end
```

Comment:
The `vim_base_uri` and `vimeo_access_token` methods here can be changed into constants. From what I can see, it looks like the reason you made them methods was to use them with the `helper_method` in the `ApplicationController`. However, I don't see you utilize this functionality in your views and instead only see them called in this module, so they should instead be constants.